internal/commands/vaultsecrets/gatewaypools/create.go

```diff
@@ -207,6 +209,7 @@ func createRun(opts *CreateOpts) error {
 type gatewayCreds struct {
 	ProjectID    string `json:"project_id,omitempty"`
 	ResourceName string `json:"resource_name,omitempty"`
+	ResourceID   string `json:"resource_id,omitempty"`
```

Comment:
I don't think this belongs in the creds.json file

Reply:
Nor does ResourceName, actually.

Reply:
Thanks, removed in https://github.com/hashicorp/hcp/pull/195/commits/edbdc6794ab85de1877652c3b5024c37d6b711aa